open-company/open-company-web
# Scss

card: https://trello.com/c/lXRPb33Y

test:
- install sass if you haven't already (http://sass-lang.com/install)
- start the local development server (`lein figwheel!` and `lein scss!`)
- check in the sass shell that it compile the first time w/o errors
- open a company page
- check that everything looks fine
- deploy this branch after pulling infrastructure (there are 2 new commit already on master)
- load a company page from the remote server and check that the css looks fine
- make sure that main.css is loaded instead of the old style.css
- check the output of travis-ci, it should now check the sass output as final task
